cookie.ts

```diff
@@ -62,7 +61,7 @@ export function formatCookieString (key : string, value : any, options : CookieO
  */
 export function getCookie (key : string) : any
 {
-    const matcher = new RegExp(`; ${escapeStringRegexp(key)}=([^;]+)`);
+    const matcher = new RegExp(`; ${escapeString(key)}=([^;]+)`);
```

Comment:
You should leave the old name, because it's important **what for** we are escaping.